cli/bench/space.go

```diff
@@ -54,3 +71,23 @@ func dropBenchmarkSpace(tarantoolConnection *tarantool.Connection) error {
 	}
 	return nil
 }
+
+// fillBenchmarkSpace fills benchmark space with a PreFillingCount number of records using connectionPool for fast filling
```

Comment:
The string length is greater than 100. Why this is bad described in the previous review.